### PR TITLE
Typography Section

### PR DESCRIPTION
**Pull request by jfgodin**
Removed the old PR related to this as the branch was too old for merging in to Main.

### CODE REVIEW

File: docs/design-language.md [hunk 1]
```diff
@@ -330,3 +330,70 @@ There are four types of notifications available to use:
...
+    <p class="caption">Caption text (0.875rem/14px)</p>
+</div><br />
+
+### Font Stack
+
+Our current font stack is <em>"Segoe UI, Helvetica Neue, Roboto, Helvetica, Arial, sans-serif"</em>. **<a href="https://learn.microsoft.com/en-us/typography/font-list/segoe-ui" target="_blank">Segoe UI</a>** is installed by default on Windows. MacOS/iOS will take advantage of Helvitca Neue and Roboto is Android's default font. **Font sizes rely on the browser default font-size of 16px. This value should not be changed.**
+
+### Headings
+
+Headings are the largest text on the screen and are used for titles. They help organize content and demonstrate importance using a hierarchy that ranges from 1-6. For more information regarding headings please see **<a href="https://www.w3.org/WAI/tutorials/page-structure/headings/" target="_blank">W3C</a>** and **<a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements" target="_blank">MDN Web Docs</a>**.
```
Comment (garrettdigital) [inline docs/design-language.md:362]: I'd maybe reword this a bit to tell people about the best practices/proper use information on the reference sites. eg: "For more information regarding headings and the best practices for usage, please refer to W3C and MDN Web Docs."

File: docs/design-language.md [hunk 2]
```diff
@@ -330,3 +330,70 @@ There are four types of notifications available to use:
...
+
+#### Considerations when using Headings
+
+-   Do not alter the weight of headings using CSS or HTML tags.
```
Comment (garrettdigital) [inline docs/design-language.md:366]: Should we also tell people to not alter the size along with the weight?

File: docs/design-language.md [hunk 3]
```diff
@@ -330,3 +330,70 @@ There are four types of notifications available to use:
...
+-   Avoid skipping heading levels. Maintain an appropriate logical hierarchy for each section of an app.
+-   In most cases, such as in Studio Web, the first heading in a Panel component should be an H2 (H1 is reserved for the app title).
```
Comment (garrettdigital) [inline docs/design-language.md:368]: Do we still need this if we are referring people to the W3C and MDN sites for best practices? Plus not all layouts will have panels.

**Review by garrettdigital — CHANGES_REQUESTED** [review]

**Review by eric-parsons — APPROVED** [review]

File: docs/design-language.md (hunk 2 above)
Comment (jfgodin) [reply docs/design-language.md:366]: Yeah!

File: docs/design-language.md (hunk 1 above)
Comment (jfgodin) [reply docs/design-language.md]: Updated.

File: docs/design-language.md (hunk 2 above)
Comment (jfgodin) [reply docs/design-language.md]: Updated.

File: docs/design-language.md (hunk 3 above)
Comment (jfgodin) [reply docs/design-language.md]: Updated. Added a point about multiple H1 tags. I feel it's important enough to add there as not everyone will click on the external link to read more.

File: docs/design-language.md (hunk 2 above)
Comment (garrettdigital) [reply docs/design-language.md]: A new thought. We should advise people to not chose a heading based on the size that they want (because that could change in future MUI updates) and that they should chose a heading level based on it's the heirarchy of the content.

File: docs/design-language.md (hunk 3 above)
Comment (garrettdigital) [reply docs/design-language.md]: I like it.

File: docs/design-language.md (hunk 2 above)
Comment (jfgodin) [reply docs/design-language.md]: Updated.

**Review by garrettdigital — APPROVED** [review]
